app/controllers/customers_controller.rb

```diff
@@ -0,0 +1,16 @@
+class CustomersController < ApplicationController
+  def index
+    @customers = Customer.request_query(params, Customer)
+
+    if @customers.empty?
```

Comment:
Generally an empty set of data isn't an error, but rather it would just return an empty array indicating that there was no data available